dvc/commands/experiments/show.py

```diff
@@ -426,11 +426,12 @@ def show_experiments(
             subset=subset,
         )
         td.drop_duplicates("rows", subset=subset)
-        td.column("Experiment")[:] = [
-            # remove tree characters
-            str(x).encode("ascii", "ignore").strip().decode()
-            for x in td.column("Experiment")
-        ]
+        if "Experiment" in td:
+            td.column("Experiment")[:] = [
+                # remove tree characters
+                str(x).encode("ascii", "ignore").strip().decode()
+                for x in td.column("Experiment")
+            ]
```

Comment:
Would it be possible to never add a tree character in `prepare_exp_id` if `pcp` is enabled?